lib/resources/r4/care_team.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: CareTeam
+field_name_base_url: http://hl7.org/fhir/R4/careteam-definitions.html#CareTeam
+fields:
+- name: patient
+  required: 'Yes'
+  type: Reference
+  description: Indicates the patient who the sensitivity is for.
+  example: |
+    {
+      "patient": {
+        "reference": "Patient/631923"
+      }
+    }
+  url: https://hl7.org/fhir/r4/careteam-definitions.html#CareTeam.patient
+
+- name: encounter
+  required: 'No'
+  type: Reference
+  description: Indicates the encounter when the allergy or intolerance was asserted.
+  example: |
+    {
+      "encounter": {
+        "reference":"Encounter/1693915"
+      }
+    }
+  url: https://hl7.org/fhir/r4/careteam-definitions.html#CareTeam.encounter
+
+- name: category
+  required: 'No'
+  type: List of CodeableConcept
```

Comment:
required and type keys do not appear to be needed in your .md - you are only adding a terminology table instead of a body fields table

Reply:
Added the required body fields and updated the terminology table. https://github.com/cerner/fhir.cerner.com/pull/470/commits/743b57306d7803dca473eb256c61982ec9c0ff62

Reply:
the way to address this comment is by removing required: and type: keys

those keys are needed when rendering body fields tables, but not needed when rendering terminology tables

Reply:
+1

Reply:
Addressed. https://github.com/cerner/fhir.cerner.com/pull/470/commits/60de10bf8259905011fb638e99b4f1607031c126#diff-c37b2a12be12c28b23605fef2eacc3ceL17-L18